indico/web/client/styles/modules/_papers.scss

```diff
@@ -46,6 +46,14 @@
     label {
         font-weight: normal;
     }
+
+    .affiliation {
+        font-size: 0.8em;
+
+        .warning-text {
+            color: #e99e18;
```

Comment:
You have this color defined in the `_palette.scss` as `$yellow`

Reply:
okay, Added :+1: